frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -190,6 +191,13 @@ const Sidebar: React.FC<SidebarProps> = ({ isCollapsed, setIsCollapsed }) => {
                     to: "/details-settings",
                     tiers: ["Basic + Financial Assistant", "Custom + Financial Assistant", "Premium + Financial Assistant"],
                     roles: ["admin", "user"]
+                },
```

Comment:
@egdagger reports should be access only platform admin tab would be a new role to be created 

we can capture this in a new task